www/js/controllers.js

```diff
@@ -194,11 +194,12 @@ angular.module('app.controllers', [])
       });
     }])
 
-  .controller('profileCtrl', ['$scope', '$stateParams', // The following is the constructor function for this page's controller. See https://docs.angularjs.org/guide/controller
+  .controller('profileCtrl', ['$scope', '$stateParams', 'Auth',// The following is the constructor function for this page's controller. See https://docs.angularjs.org/guide/controller
     // You can include any angular dependencies as parameters for this function
     // TIP: Access Route Parameters for your page via $stateParams.parameterName
-    function ($scope, $stateParams) {
-
+    function ($scope, $stateParams, Auth) {
+        Auth.user=this;
+            getDisplayName();
```

Comment:
You can use the `console.log` inside of this controller to print information to the console for development purposes. If you add the `console.log(Auth.$getAuth())` and refresh the page, the user information should be printed to the developer tools console.